### ISSUE TITLE
/etc/yum.repos.d/xCAT-*.repo point to master after controller deployment (affects compute nodes as well)

### ISSUE THREAD

**Issue by rispoli** [clustervision/trinity#597]
After a controller has been deployed it only has two repositories enabled, the local ones and they both point to the master used to deploy it (192.168.1.254).
These settings are then copied into the images which will show the same problem.

If we want to keep the local repositories it'd be better to replace the master's IP with a reference to the controller,:
```
sed -i -e 's/192.168.1.254/controller.cluster/' /etc/yum.repos.d/xCAT-*.repo
```

The normal distribution-provided repositories have to be enabled as well, at least on the controller.

See #472, #473 .

```
# /trinity/report 
**Version:** r7_login_image 936bccb
**Environment:** 150373
```

**Comment by hansthen** [comment]
Duplicate of #473.

**Comment by hansthen** [comment]
Fixed in 3b471220df (R7)
